### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by besnardf**
Solves issue https://github.com/eduayme/RescueApp/issues/145

### CODE REVIEW

**Review by eduayme — APPROVED** [review]

File: resources/lang/fr/profile_lost_person.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+return [
+    'development_disorder'            => 'Trouble du développement',
+    'alzheimer_or_other_dementias'    => 'Alzheimer ou autres démences',
+    'mental_or_psychological_illness' => 'Troubles psychologiques',
+    'autolithic_behaviors'            => 'Comportements autolithiques',
+    'hiker'                           => 'Randonneur',
+    'collector'                       => 'Collectionneur'
```
Comment (eduayme) [inline resources/lang/fr/profile_lost_person.php:9]: A comma is missing after 'Collectionneur'

**Review by eduayme — CHANGES_REQUESTED** [review]
2 mistakes found. If solved it will be merge to the project

File: resources/lang/fr/forms.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    'region'                                    => 'Région',
+    'begin_day'                                 => 'Jour de début',
+    'end_day'                                   => 'Jour de fin',
+    'day'                                       => 'Jour',
+    'date_time'                                 => 'Date et heure',
+    'images'                                    => 'Images',
+    'chose_option'                              => 'Choisir une option',
+    'alertant'                                  => 'Alerte',
+    'is_the_lost_person'                        => 'Est la personne recherchée',
+    'not_is_the_lost_person'                    => 'N\'est pas la personne recherchée,
```
[the remark this answers is not in the document]
Comment (besnardf) [reply resources/lang/fr/forms.php]: Fixed in https://github.com/eduayme/RescueApp/pull/148/commits/794fd5ca5d8261cc804a9d362527f4f918781090

File: resources/lang/fr/profile_lost_person.php (hunk 1 above)
Comment (besnardf) [reply resources/lang/fr/profile_lost_person.php]: Fixed in https://github.com/eduayme/RescueApp/pull/148/commits/794fd5ca5d8261cc804a9d362527f4f918781090

**Comment by besnardf** [comment]
@eduayme I translated the project to French as part of the Hacktoberfest.
Can you please add the Hacktoberfest topic to the whole project so that it can count towards my goal of 4 PR during this month ?

**Review by eduayme — APPROVED** [review]
Everything nice now! Thanks :)

**Pull request merged**